src/testing/docker/travis_docker_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+docker build -t fossology/test . 
+container_id=`docker run -d fossology/test`
+ip=`docker inspect --format '{{ .NetworkSettings.IPAddress }}' $container_id | awk "NR==1{print;exit}"`
+docker ps | grep -q fossology/test
+curl -L -I http://127.0.0.1:8081/repo/ | grep -q "200 OK"
```

Comment:
Please check whether the page really appears. E.g. grep for some text on the homepage.

If the composer installation fails Apache still delivers an empty page successfully.